main.go

```diff
@@ -0,0 +1,18 @@
+package main
+
+import (
+	"log"
+
+	"github.com/rivo/tview"
+	"github.com/ronaudinho/drt/internal/ui"
+)
+
+func main() {
+	app := tview.NewApplication()
+
+	err := ui.MainPage(app)
+
+	if err != nil {
```

Comment:
nit/style: for single err that is checked directly, prefer
```
if err := someFunc(); err != nil {
  // do something
}
```

Reply:
updated